internal/core/services/operation_manager/operation_manager.go

```diff
@@ -31,9 +31,12 @@ import (
 	"github.com/topfreegames/maestro/internal/core/entities/operation"
 	"github.com/topfreegames/maestro/internal/core/operations"
 	"github.com/topfreegames/maestro/internal/core/ports"
+	"github.com/topfreegames/maestro/internal/core/ports/errors"
+	"go.uber.org/zap"
 )
 
 type OperationManager struct {
+	operationCancelFunctions        map[string]map[string]context.CancelFunc
```

Comment:
@luizmiranda7 this is a `map[schedulerId]map[operationId]context.CancelFunc`, right? WDYT about using some type alias for more readability here?

Reply:
Changed